federation/cmd/federation-apiserver/app/server.go

```diff
@@ -196,9 +194,15 @@ func Run(s *options.ServerRunOptions) error {
 		sets.NewString("watch", "proxy"),
 		sets.NewString("attach", "exec", "proxy", "log", "portforward"),
 	)
+	genericConfig.RESTOptionsGetter = &kubeapiserver.RESTOptionsFactory{
```

Comment:
this feels strange that we have generic etcd options, pass it through the ApplyTo calls and then only use it here in each apiserver to create a non-generic rest options factory.